packages/matplotlib/patches/fix-threading.patch

```diff
@@ -0,0 +1,67 @@
+diff --git a/lib/matplotlib/backend_bases.py b/lib/matplotlib/backend_bases.py
+--- a/lib/matplotlib/backend_bases.py
++++ b/lib/matplotlib/backend_bases.py
+@@ -1993,6 +1993,7 @@ class FigureCanvasBase:
+ 
+     def draw(self, *args, **kwargs):
+         """Render the `.Figure`."""
++        pass
```

Comment:
Is this accidentally left over or is there some mysterious python shenanigans?

Reply:
I think it is accidental but can't figure out why it doesn't throw error... I just fixed it passing by.

Reply:
This is not fixed upstream and they have multiple use of `pass` in this file. I'll proposes a PR upstream.

Reply:
Empty function with docstring seems [allowed by the Python's grammar](https://stackoverflow.com/questions/17735170/why-does-python-allow-an-empty-function-with-doc-string-body-without-a-pass).

Reply:
Anyway, this is confusing in this PR. So, upstream PR + removed from this PR if you agree.

Reply:
Sounds good to me. Removing from this PR also means one less patch to worry about when we update later.